src/Lykke.Job.SiriusCashoutProcessor/Workflow/CommandHandlers/CashoutCommandHandler.cs

```diff
@@ -65,13 +65,18 @@ public async Task<CommandHandlingResult> Handle(StartCashoutCommand command, IEv
                 Signature = signature
             });
 
-            var techData = result.ResultCase == WithdrawalExecuteResponse.ResultOneofCase.Body
-                ? result.Body.Withdrawal.ToJson()
-                : result.Error.ToJson();
-
-            _log.Info("Cashout sent to sirius", context: $"result: {techData}");
-
-            return CommandHandlingResult.Ok();
+            if (result.ResultCase == WithdrawalExecuteResponse.ResultOneofCase.Error)
+            {
+                _log.Error("Cashout to Sirius failed", context: $"result: {result.Error.ToJson()}");
+                
+                return CommandHandlingResult.Fail(TimeSpan.FromSeconds(10));
```

Comment:
In the case of `INVALID_PARAMETERS` or `INVALID_ADDRESS` retries doesn't make any sense, so we can just fail the message without retries. If it's possible in Lykke.Cqrs